TicketBookingBackend/TicketBookingAPI/TicketBookingDomain/Ticket.cs

```diff
@@ -6,7 +6,7 @@ public class Ticket
         public int BusId { get; set; }
         public Bus Bus { get; set; }
         public int UserId { get; set; }
-        public User User { get; set; }
+        public Users User { get; set; }
```

Comment:
The class name should be singular and not plural

Reply:
resolved